src/SessionMonitor.ts

```diff
@@ -153,36 +150,33 @@ export class SessionMonitor {
                     this._checkSessionIFrame.start(session.session_state);
 
                     if (session.sid === this._sid) {
-                        this._logger.debug("_callback: Same sub still logged in at OP, restarting check session iframe; session_state:", session.session_state);
+                        logger.debug("same sub still logged in at OP, restarting check session iframe; session_state", session.session_state);
                     }
                     else {
-                        this._logger.debug("_callback: Same sub still logged in at OP, session state has changed, restarting check session iframe; session_state:", session.session_state);
+                        logger.debug("same sub still logged in at OP, session state has changed, restarting check session iframe; session_state", session.session_state);
                         this._userManager.events._raiseUserSessionChanged();
                     }
                 }
                 else {
-                    this._logger.debug("_callback: Different subject signed into OP:", session.sub);
+                    logger.debug("different subject signed into OP", session.sub);
                 }
             }
             else {
-                this._logger.debug("_callback: Subject no longer signed into OP");
+                logger.debug("subject no longer signed into OP");
             }
 
             if (raiseEvent) {
                 if (this._sub) {
-                    this._logger.debug("_callback: SessionMonitor._callback; raising signed out event");
```

Comment:
why no more logging, generally we should log more than less...

Reply:
I'll add an else block logging that it's explicitly not raising an event, otherwise these lines weren't adding any useful info since raising the event already prints out everything you need to know, e.g.


```
[SessionMonitor] _callback: different subject signed into OP
[Event("User signed out"): raise
```

```
[SessionMonitor] _callback: same sub still logged in at OP, restarting check session iframe
                       (no further logs from this method)
```